### PR TITLE
Webhook proxy

### PR DESCRIPTION
**Pull request by JasonEtco**
This PR basically does the whole thing. It absorbs @bkeepers [experiment project](https://github.com/bkeepers/webhook-proxy), fixes up all the routes and integrates the UI.

It kinda works. It's very hard to tell, because Heroku has timeout-problems when it comes to Server-Sent-Events. It's hard to know what part of the system is borking after about a minute. Sending events within the first few seconds of a visit to the site (on a new channel or since the last deploy) works.

@bkeepers - we can merge this in now, you're welcome to peer through the changes (only `index.js` is really important to you). I tried moving the EventSource route to be `/:channel/stream` for more reliable routes.

I've deleted all of the authentication stuff; that can be for the future if we ever decide to re-add it.

Also FWIW, web sockets are objectively better and way more reliable 😜

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,60 +6,54 @@
   "license": "ISC",
   "repository": "https://github.com/probot/webhooks.git",
   "scripts": {
-    "start": "concurrently \"npm run start-dev\" \"npm run watch-ui\"",
-    "start-dev": "nodemon --ignore src/ --ignore dist/ --ignore logs.json ./node_modules/probot/bin/probot-run ./index.js",
-    "start-prod": "probot run ./index.js",
+    "start": "node index.js",
+    "start-dev": "concurrently \"nodemon --ignore src/ ./index.js\" \"webpack -u\"",
```
Comment (bkeepers) [inline package.json]: `webpack -w`? I get webpack's usage info printed when I run this.

Comment (JasonEtco) [reply package.json]: People make typos sometimes. This is one of those times 😂

**Comment by JasonEtco** [comment]
@bkeepers regarding your deployment woes - I haven't had that issue. I've done nothing special to deploy, I just hooked it up to the repo and hit **Deploy Branch**. You can try disabling the NPM_MODULES cache in Heroku, there may be some artifacts from the old deploy?

**Comment by bkeepers** [comment]
👍 deployed: http://github-webhook-proxy.herokuapp.com/

**Pull request merged**